src/Notes/AfterCampaignMigration.php

```diff
@@ -0,0 +1,79 @@
+<?php
+declare( strict_types=1 );
+
+namespace Automattic\WooCommerce\GoogleListingsAndAds\Notes;
+
+use Automattic\WooCommerce\Admin\Notes\Note as NoteEntry;
+use Automattic\WooCommerce\GoogleListingsAndAds\Ads\AdsAwareTrait;
+use Automattic\WooCommerce\GoogleListingsAndAds\HelperTraits\Utilities;
+use Automattic\WooCommerce\GoogleListingsAndAds\Ads\AdsAwareInterface;
+use Automattic\WooCommerce\GoogleListingsAndAds\PluginHelper;
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * Class AfterCampaignMigration
+ *
+ * @package Automattic\WooCommerce\GoogleListingsAndAds\Notes
+ */
+class AfterCampaignMigration extends AbstractNote implements AdsAwareInterface {
```

Comment:
💅  IMHO (and I'm aware they are plenty of these in the codebase) comment a class with `// class [className]` is redundant since we already see it in the PHP syntax. I think it would be more helpful to explain when this Notice is appearing in the Inbox and why giving context for future devs.